src/frontend/src/components/landingPage.ts

```diff
@@ -0,0 +1,58 @@
+import { I18n } from "$src/i18n";
+import { html, TemplateResult } from "lit-html";
+import { navigationLink } from "./footer";
+import { githubIcon, icLogo, questionIcon } from "./icons";
+import copyJson from "./landingPage.json";
+
+/**
+ * Landing page template
+ *
+ * It is a component with split panes left and right in desktop view.
+ * To the left there is some static content, and to the right there is a slot.
+ *
+ * In movile view, the static content is below the slot.
+ *
+ */
+export const landingPage = ({
+  slot,
+}: {
+  slot: TemplateResult;
+}): TemplateResult => {
+  const i18n = new I18n();
+  const copy = i18n.i18n(copyJson);
+
+  return html` <main class="c-landingPage">
+    <div class="c-landingPage__logo">
+      <div class="c-logo">${icLogo}</div>
+    </div>
+    <section class="c-landingPage__right">
```

Comment:
semantic elements are supposed to have either or an aria-label

Reply:
either what?

I added an `aria-label` makes sense to me.

Reply:
either a title (h1, h2, h3...) or aria-labael. sorry

Reply:
Ahh, ok. In that case, yes, the section will have a title already. The component inside always has an `h1`.